integration/docker/entrypoint.sh

```diff
@@ -89,8 +90,10 @@ function mountAlluxioRootFSWithFuseOption {
   fi
 
   # Unmount first if cleanup failed and ignore error
-  ! integration/fuse/bin/alluxio-fuse unmount /alluxio-fuse
-  exec integration/fuse/bin/alluxio-fuse mount -n ${fuseOptions} /alluxio-fuse /
+  ! mkdir -p ${MOUNT_POINT}
+  ! umount ${MOUNT_POINT}
+  #! integration/fuse/bin/alluxio-fuse unmount ${MOUNT_POINT}
+  exec integration/fuse/bin/alluxio-fuse mount -n ${fuseOptions} ${MOUNT_POINT} /
```

Comment:
I'm not sure if this change will fix this issue.
I encounter same error when alluxio-fuse daemon is kill by SIGKILL. In this situation, the `preStop` hook will not be executed.
And when alluxio-fuse daemon trying to restart, k8s failed to mount the `alluxio-fuse-mount`, and report the error: `Transport endpoint is not connected`. I suspect we don't have a change to execute this script, container will be failed in start phrase.

One workaround is to use sidecar to monitor the this mount point. If the mount point is broken, umount it in sidecar. But I think the more official way is using PV/PVC to deploy alluxio in k8s. And let k8s to manage the storage life-cycle.

Reply:
This sould be also used with `mountPath: /mnt` in `integration/kubernetes/helm-chart/alluxio/values.yaml` The pod is not mounted with the alluxio fuse mount path, instead its parent directory.